tagtransform-c.cpp warning

As one of our goals is to have no warnings, this is a bug.

```
/home/pnorman/osm/osm2pgsql/tagtransform-c.cpp: In member function ‘virtual bool c_tagtransform_t::filter_rel_member_tags(const taglist_t&, const osmium::memory::Buffer&, const rolelist_t&, int*, int*, int*, int*, const export_list&, taglist_t&, bool)’:
/home/pnorman/osm/osm2pgsql/tagtransform-c.cpp:378:25: warning: ‘flags’ may be used uninitialized in this function [-Wmaybe-uninitialized]
             if (!(flags & FLAG_POLYGON)) {
                  ~~~~~~~^~~~~~~~~~~~~~~
```

Produced with gcc 6.3.0

--- a/tagtransform-c.cpp
+++ b/tagtransform-c.cpp
@@ -367,7 +367,7 @@ bool c_tagtransform_t::filter_rel_member_tags(
             // Copy the list identified outer way tags over to the relation
             // filtering for wanted tags on the way.
             bool filter;
-            int flags;
+            int flags = 0;
             for (const auto &poly_tag : poly_tags) {
                 if (check_key(infos, poly_tag.key.c_str(), &filter, &flags,
                               false)) {

